### ISSUE TITLE
How do I make buttons out of links?

### ISSUE THREAD

**Issue by LeanneCotter** [MODatUniSA/mod-website#1]
Here’s how to turn a normal link into a button on our website:

`<a href="#">Subscribe</a>`

Change that to:

`<a href="#" class="btn btn-outline-secondary">Subscribe</a>`

If you’d like a large button:

`<a href="#" class="btn btn-lg btn-outline-secondary">Subscribe</a>`

And for different colours:

`btn btn-outline-primary` (blue)
`btn btn-outline-secondary` (grey)
`btn btn-outline-success` (green)
`btn btn-outline-info` (light blue)
`btn btn-outline-warning` (orange)
`btn btn-outline-danger` (red)

So basically just add `class="btn btn-outline-secondary"` to any links to make them a button.

**Comment by sighmon** [comment]
I should probably have got you to post this as a wiki post rather than an issue.
